WordPress/Classes/ViewRelated/Plans/PurchaseButton.swift

```diff
@@ -104,7 +103,7 @@ class PurchaseButton: RoundedButton {
                     self.cornerRadius = self.bounds.height / 2
                     self.titleLabel?.alpha = 0
                     
-                    self.layoutIfNeeded()
+                    self.superview?.layoutIfNeeded()
```

Comment:
**Amazing** gif usage!

So, this is the only way I can find to prevent the animation glitch we were seeing (PurchaseButton's superview is coloured in green):

![purchase-broken](https://cloud.githubusercontent.com/assets/4780/14881135/9aecb558-0d2a-11e6-822a-07005d83c61c.gif)

Without telling the superview to lay out, it doesn't seem to update its size correctly to contain the button and keep it centered.